src/main/java/com/threeamigos/music/yetanotherspotify/controller/SongArtistController.java

```diff
@@ -0,0 +1,45 @@
+package com.threeamigos.music.yetanotherspotify.controller;
+
+import com.threeamigos.music.yetanotherspotify.model.Artist;
+import com.threeamigos.music.yetanotherspotify.model.Song;
+import com.threeamigos.music.yetanotherspotify.service.ArtistService;
+import com.threeamigos.music.yetanotherspotify.service.SongService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/")
```

Comment:
maybe change the path?

Reply:
What path do you suggest?

Reply:
"/SongArtist"

Reply:
Or change the name of the controller to something more suitable.
Or a controller dedicated to artists and another for songs  🗡️

Reply:
Resolved. Now the path is /SongArtist.